com.github.robertsanseries.ciano.json

```diff
@@ -0,0 +1,142 @@
+            ],
+            "cleanup": [
+                "/bin",
+                "/include",
+                "/share/vala",
+                "/share/gir-1.0",
+                "/share/applications",
+                "/lib/pkgconfig",
+                "/lib/girepository-1.0",
+                "/lib/*.la"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://launchpad.net/granite/0.4/0.4.1/+download/granite-0.4.1.tar.xz",
+                    "sha256": "8bb411523e6cd5685c928dbfe08893892189cedf9fad5f537aa76182032cf09c"
+                }
+            ]
+        },
+        {
+          "name": "imagemagick",
+          "config-opts": [
+            "--enable-static=no",
+            "--disable-docs",
+            "--disable-deprecated",
+            "--without-autotrace",
+            "--without-bzlib",
+            "--without-djvu",
+            "--without-dps",
+            "--without-fftw",
+            "--without-fontconfig",
+            "--without-fpx",
+            "--without-freetype",
+            "--without-gvc",
+            "--without-jbig",
+            "--without-jpeg",
+            "--without-lcms",
+            "--without-lzma",
+            "--without-magick-plus-plus",
+            "--without-openexr",
+            "--without-openjp2",
+            "--without-pango",
+            "--without-raqm",
+            "--without-tiff",
+            "--without-webp",
+            "--without-wmf",
+            "--without-x",
+            "--without-xml",
+            "--without-zlib"
+          ],
+          "sources": [
+            {
+              "type": "archive",
+              "url": "http://mirror.checkdomain.de/imagemagick/ImageMagick-7.0.7-21.tar.xz",
+              "sha256": "3b2783a845279f76a4250f2c3caf04af22766e2d5f084bde589ebd73de6a0019"
+            }
+          ]
+        },
+        {
+            "name": "ffmpeg",
+            "cleanup": [ 
+                "/include", 
+                "/lib/pkgconfig", 
+                "/share/ffmpeg/examples" 
+            ],
+            "config-opts": [
+                "--enable-shared",
+                "--disable-static",
+                "--disable-ffserver",
+                "--disable-ffplay",
+                "--disable-ffprobe",
+                "--disable-doc",
+                "--disable-network"
+            ],
+            "sources": [
+                {
+                  "type": "archive",
+                  "url": "https://www.ffmpeg.org/releases/ffmpeg-3.4.1.tar.xz",
+                  "sha256": "5a77278a63741efa74e26bf197b9bb09ac6381b9757391b922407210f0f991c0"
+                }
+            ]
+        },
+        {
+            "name": "ciano",
+            "buildsystem": "meson",
+            "builddir": true,
+            "sources": [
+                {
+                  "type":"git",
+                  "url":"https://github.com/robertsanseries/ciano.git"
```

Comment:
This needs to be a stable release. https://github.com/robertsanseries/ciano/archive/0.1.9.tar.gz